js/packages/eyes-api/src/input/CheckSettings.ts

```diff
@@ -89,6 +89,7 @@ export type Target<TElement, TSelector> = {
     scrollRootElement?: ElementReference<TElement, TSelector>,
   ): CheckSettingsFluent<TElement, TSelector>
   shadow(selector: SelectorReference<TSelector>): CheckSettingsFluent<TSelector>
+  webview(option: string | boolean | null): CheckSettingsFluent<TElement, TSelector>
```

Comment:
It actually should be optional
```suggestion
  webview(option?: string | boolean): CheckSettingsFluent<TElement, TSelector>
```

Reply:
Ah, I see